ehsm_kms_service/ehsm_kms_server.js

```diff
@@ -258,5 +258,20 @@ app.post('/ehsm', function (req, res) {
   }
 })
 
-app.listen(port, () => {})
+const  getIPAdress = () => {
+  var interfaces = require('os').networkInterfaces();　　
+  for (var devName in interfaces) {　　　　
+      var iface = interfaces[devName];　　　　　　
+      for (var i = 0; i < iface.length; i++) {
+          var alias = iface[i];
+          if (alias.family === 'IPv4' && alias.address !== '127.0.0.1' && !alias.internal) {
+              return alias.address;
+          }
+      }　　
+  }
+}
+
+app.listen(port, () => {
+  console.log(`ehsm_ksm_service application listening at ${getIPAdress()}:${port}`)
```

Comment:
you need to print out the target ip that who connected to the server, not the server's IP

Reply:
This is the log printed once when the server is started, which is used to display the IP address of the currently running nodejs server。